lib/data_raw.lua

```diff
@@ -1,17 +1,42 @@
 local data_raw = {}
+data_raw.removed = {}
```

Comment:
should we rename this to `removed_or_hidden`?

Reply:
Hmm... that implies that it gets everything `.hide()` is called on... which might actually be the right thing to do anyway.